tests/dom-utils.test.js

```diff
@@ -2,14 +2,33 @@ const {
   hideElement,
   showElement,
   toggleVisibleElement,
+  sortChildren,
 } = require('../public/scripts/dom-utils.js');
 
+const orderedData = [
+  { 'order-alpha': 1, getAttribute: (order) => 1 },
+  { 'order-alpha': 2, getAttribute: (order) => 2 },
+  { 'order-alpha': 3, getAttribute: (order) => 3 },
+];
+const testData = [
+  { 'order-alpha': 3, getAttribute: (order) => 3 },
+  { 'order-alpha': 2, getAttribute: (order) => 2 },
```

Comment:
To avoid a false negative in the test if the implementation reverse ordering you could invert the order of these two items:

```suggestion
  { 'order-alpha': 2, getAttribute: (order) => 2 },
  { 'order-alpha': 3, getAttribute: (order) => 3 },
```